src/SDK.ts

```diff
@@ -42,8 +42,9 @@ import { CustomContextData } from "./Utils/CustomContextData";
 
 export default class SDK implements ISDK {
   private static defaultConfiguration: ISDKConfiguration = {
-    getChatTokenRetryCount: 35,
+    getChatTokenRetryCount: 10,
     getChatTokenTimeBetweenRetriesOnFailure: 10000,
+    getChatTokenRetryOn429: false,
```

Comment:
are we exposing this config to allow the customer set on-off ?or it will be hardcoded as well?,